k8s.io/configmap-nginx.yaml

```diff
@@ -232,6 +232,7 @@ data:
         location / {
           rewrite ^/api-review$      https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md redirect;
           rewrite ^/bot-commands$    https://prow.k8s.io/command-help redirect;
+          rewrite ^/calendar$        https://calendar.google.com/calendar/embed?src=calendar%40kubernetes.io redirect;
```

Comment:
IIRC this calendar is no longer the "official' one... https://www.k8s.dev/calendar is the new one

cc @mrbobbytables

Reply:
That is the right calendar, but we do want folks to goto the the k8s.dev site instead. It has more friendly views and doesn't assume PST

Reply:
good point